app/ts/components/App.tsx

```diff
@@ -28,7 +28,7 @@ export function App() {
 						<>
 							<Import {...state} />
 							{state.bundle.value ? <Transactions {...state} /> : null}
-							<Configure {...state} />
+							{/* <Configure {...state} /> */}
```

Comment:
why not removed?

Reply:
Configure has a bug and was going to fix in next PR.